public/content/contributing/adding-resources/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Adding Resources
+description: The policy we use when adding resources to ethereum.org
+lang: en
+---
+
+# Adding Resources {#adding-resources}
+
+We want to make sure we list the best resources possible while keeping users safe and confident.
+
+Anyone is free to suggest new resources to add to the resource dashboard on ethereum.org, currently found at [ethereum.org/resources](/resources/).
+
+**Please only suggest new additions for this page.**
+
+Although we welcome new additions, the current resources were chosen based on an experience we're trying to create for our users. These are based on some of our design principles:
+
+- _Inspirational_: anything on ethereum.org should offer something new to users
+- _A good story_: what's listed should provide an "aha" moment
```

Comment:
Not saying we removed it, but this seems less true in dashboards than elsewhere. Perhaps because dashboards is a more 'advanced page'